### PR TITLE
fixing the run title check

### PR DESCRIPTION
**Pull request by kaadhina**

### CODE REVIEW

File: Tasks/Gradle/Gradle.ps1 [hunk 1]
```diff
@@ -162,14 +167,19 @@ if($publishJUnitResultsFromAntBuild)
...
-		}
-		else
+	$runTitleMemberExists = CmdletHasMember "RunTitle"
+	if($runTitleMemberExists)
+	{
+		Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle
+	}
+	else
+	{
+		if(!([string]::IsNullOrWhiteSpace($testRunTitle)))
```
Comment (allendm-msft) [inline Tasks/Gradle/Gradle.ps1]: use the boolean

File: Tasks/ANT/ant.ps1 [hunk 2]
```diff
@@ -179,14 +184,19 @@ if($publishJUnitResultsFromAntBuild)
...
-		}
-		else
+	$runTitleMemberExists = CmdletHasMember "RunTitle"
+	if($runTitleMemberExists)
+	{
+		Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle
+	}
+	else
+	{
+		if(!([string]::IsNullOrWhiteSpace($testRunTitle)))
```
Comment (allendm-msft) [inline Tasks/ANT/ant.ps1]: use the boolean

File: Tasks/Maven/maven.ps1 [hunk 3]
```diff
@@ -101,13 +106,18 @@ Write-Host "Running Maven..."
...
+$runTitleMemberExists = CmdletHasMember "RunTitle"
+if($runTitleMemberExists)
 {
-	PublishTestResults $publishJUnitResults $testResultsFiles
+	PublishTestResults $publishJUnitResults $testResultsFiles $testRunTitle
 }
 else
 {
-	PublishTestResults $publishJUnitResults $testResultsFiles $testRunTitle		
+	if(!([string]::IsNullOrWhiteSpace($testRunTitle)))
```
Comment (allendm-msft) [inline Tasks/Maven/maven.ps1]: here too

**Comment by allendm-msft** [comment]
what is the testing done

**Comment by allendm-msft** [comment]
as discussed share this out with the other contributors

**Comment by kaadhina** [comment]
Tested: Old agent & New agent : With & Without run title.
I've notified the other contributors.

**Comment by allendm-msft** [comment]
my previous comments were incorrect.. removed and approved.

**Pull request merged**